docs/translating/russian.md

```diff
@@ -20,12 +21,13 @@
 * sign up - **зарегистрироваться**
 * stream - **канал**
 * realm - **организация**
+* organization - **организация**
```

Comment:
Why does realm and organization have the same translation?

Reply:
Methinks in an earlier implementation of Zulip, "realm" might have been what today is referred to as "organization". As of today, you don't find "realm" anywhere in the translatable source text. You can find it in variables, though (but you don't translate variables). For example:

translations.json:
#46 Add extra emoji for members of the __realm_name__ organization.

django.po:
#372 Organization: 1%(realm_url)s
#406 1%(realm_name)s on Zulip: Your new organization details
#409 You've joined the Zulip organization 1%(realm_name)s

Thus the suggestion that "realm" = "organization". We could drop the "realm" line altogether. I kept it for reference.

Reply:
I saw that https://github.com/zulip/zulip/blob/master/docs/translating/german.md explicitly says that the term "realm" usage should be discouraged. Maybe we should do the same here.

Reply:
All right, the Germans need to explicitly discourage that because they didn't translate "realm" but rather kept it as "terminus technicus" in the German translation. I did now insert some additional comment to "realm" in this style guide for Russian.

Reply:
The story here is that "realm" is what the code calls what the UI calls an "organization".  We eventually addressed the problem by changing all the strings that mentioned "realm" to say "organization" instead except for these references in variables.

So yeah, I think the correct fix is to just not suggest a translation for `realm` at all (and we should update the other style guides accordingly).